ivy-test.el

```diff
@@ -241,6 +241,26 @@ will bring the behavior in line with the newer Emacsen."
                   (ivy--regex "(foo bar"))
                  "(\\(foo).*?(bar)")))
 
+(ert-deftest counsel-url-expand ()
+  "Test ffap expansion using counsel-url-expansions."
+  ;; no expansions defined
+  (let ((counsel-url-expansions nil))
+    (should (eql (counsel-url-expand) nil)))
+  (let ((counsel-url-expansions
+         `(("^foo$" . "https://foo.com/%s")
+           ("^issue\\([0-9]+\\)" . ,(lambda (word)
+                                      (concat "https://foo.com/issues/"
+                                              (match-string 1 word)))))))
+    ;; no match
+    (defun current-word () "foobar")
```

Comment:
Probably not a wise idea to permanently redefine this function for the sake of a test. Why not write a convenience function/macro which performs the moral equivalent of
```el
(with-temp-buffer
  (insert "foobar")
  (should (equal (counsel-url-expand) nil)))
```

Reply:
I hadn't realized that definition would escape the test environment, but indeed it does. Will fix.